Controller.php

```diff
@@ -280,4 +281,23 @@ private function getPrettyDateForSite($datetime, $period, $timezone)
 
         return $prettyDate;
     }
+
+    private function fetchSites($alert)
+    {
+        $sites = SitesManagerApi::getInstance()->getSitesWithAtLeastViewAccess();
+
+        if (!$alert || !isset($alert['id_sites'])) {
```

Comment:
maybe better check with `empty($alert['id_sites'])` also likely the first `!$alert` is not needed anyway